### PR TITLE
Beta1

### PR DESCRIPTION
**Pull request by Carreau**

### CODE REVIEW

**Comment by Carreau** [comment]
I'll push that on master later, I just open the PR for the changes to be obvious.

File: docs/source/whatsnew/version5.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ 5.x Series
+============
+
+IPython 5.0
+===========
+
+Released June, 2016
+
+IPython 5.0 now uses `promt-toolkit` for the command line interface, thus
+allowing real multi-line editing and syntactic coloration as you type.
```
Comment (takluyver) [inline docs/source/whatsnew/version5.rst]: I will write a longer description of the PTK work, along with a screenshot, before we properly release.

**Comment by takluyver** [comment]
This looks good

**Comment by Carreau** [comment]
great, I'll directly push that on master as well as the tag, and revert the release number to dev.

**Comment by Carreau** [comment]
Done. 
```
...
To ssh://github.com/ipython/ipython
   bb2c7ec..1da789b  master -> master
...
 * [new tag]         5.0.0b1 -> 5.0.0b1
...
```

**Pull request merged**